airbyte-integrations/connectors/source-greenhouse/integration_tests/expected_records.txt

```diff
@@ -69,9 +69,9 @@
 {"stream":"demographics_questions","data":{"translations":[{"name":"q2","language":"en"}],"required":false,"name":"q2","id":4000715003,"demographic_question_set_id":4000197003,"answer_type":"multi_value_multi_select","active":true},"emitted_at":1660156527375}
 {"stream":"demographics_questions","data":{"translations":[{"name":"question1","language":"en"}],"required":false,"name":"question1","id":4000716003,"demographic_question_set_id":4000198003,"answer_type":"multi_value_multi_select","active":true},"emitted_at":1660156527375}
 {"stream":"demographics_questions","data":{"translations":[{"name":"question2","language":"en"}],"required":true,"name":"question2","id":4000717003,"demographic_question_set_id":4000198003,"answer_type":"multi_value_single_select","active":true},"emitted_at":1660156527389}
-{"stream":"demographics_questions","data":{"translations":[{"name":"Are you a veteran or active member of the United States Armed Forces? (select one)","language":"en"}],"required":false,"name":"Are you a veteran or active member of the United States Armed Forces? (select one)","id":4015594003,"demographic_question_set_id":4002702003,"answer_type":"multi_value_single_select","active":true},"emitted_at":1660156527389}
-{"stream":"demographics_questions","data":{"translations":[{"name":"Do you have a disability or chronic condition (physical, visual, auditory, cognitive, mental, emotional, or other) that substantially limits one or more of your major life activities, including mobility, communication (seeing, hearing, speaking), and learning? (select one)","language":"en"}],"required":false,"name":"Do you have a disability or chronic condition (physical, visual, auditory, cognitive, mental, emotional, or other) that substantially limits one or more of your major life activities, including mobility, communication (seeing, hearing, speaking), and learning? (select one)","id":4015596003,"demographic_question_set_id":4002702003,"answer_type":"multi_value_single_select","active":true},"emitted_at":1660156527389}
-{"stream":"demographics_questions","data":{"translations":[{"name":"Do you identify as transgender? (select one)","language":"en"}],"required":false,"name":"Do you identify as transgender? (select one)","id":4015598003,"demographic_question_set_id":4002702003,"answer_type":"multi_value_single_select","active":true},"emitted_at":1660156527389}
+{"stream":"demographics_questions","data":{"translations":[{"name":"Are you a veteran or active member of the United States Armed Forces?","language":"en"}],"required":false,"name":"Are you a veteran or active member of the United States Armed Forces?","id":4015594003,"demographic_question_set_id":4002702003,"answer_type":"multi_value_single_select","active":true},"emitted_at":1660156527389}
```

Comment:
I'm not sure what exactly happened here, but in the last week greenhouse might've changed their default question vernacular, but it no longer lines up with the expected records from last week. I doubled checked this against making the direct query via Postman and it does not include `(select one)` in the response so I'm confident this was a change on their end or in our test data